mipengine/tests/node/test_views.py

```diff
@@ -18,20 +18,22 @@
 
 def test_views():
     context_id = "regrEssion"
-    columns = ["subjectcode", "dataset", "subjectvisitid", "subjectvisitdate"]
+    columns = ["dataset", "age_value", "gcs_motor_response_scale", "pupil_reactivity_right_eye_result"]
     datasets = ["edsd"]
-    table_1_name = create_view.delay(context_id, str(pymonetdb.uuid.uuid1()).replace("-", ""), json.dumps(columns),
-                                     json.dumps(datasets)).get()
+    pathology = "tbi"
+    table_1_name = create_view.delay(context_id, str(pymonetdb.uuid.uuid1()).replace("-", ""), pathology, json.dumps(datasets), json.dumps(columns), "dsa").get()
```

Comment:
in function/methods with more than one argument call it with keyword arguments
     `table_1_name = create_view.delay(context_id=context_id,  ..etc...  ).get() `